cms/server/contest/templates/contest.html

```diff
@@ -59,12 +59,12 @@
             <div class="navbar-inner">
                 <div class="container">
                     <a class="brand" href="{{ contest_url() }}">{{ contest.description }}</a>
-{% if len(lang_names) > 1 %}
+{% if len(available_translations) > 1 %}
                     <p class="navbar-text pull-right">
                         <select id="lang" class="form-control btn btn-info" onchange="utils.switch_lang()">
-                            <option value=""{% if cookie_lang is None %} selected{% end %}>{{ _("Automatic (%s)") % lang_names[browser_lang] }}</option>
-                        {% for lang_code, lang_name in sorted(iteritems(lang_names), key=lambda x: x[1]) %}
-                            <option value="{{ lang_code }}"{% if cookie_lang == lang_code %} selected{% end %}>{{ lang_name }}</option>
+                            <option value=""{% if cookie_translation is None %} selected{% end %}>{{ _("Automatic (%s)") % automatic_translation.name }}</option>
```

Comment:
This "automatic" sounds quite odd - maybe default?

Reply:
I called it automatic because we render it as "Automatic (language)". Are you suggesting to change both occurrences? I'm open to that but I don't like default as then it conflicts with DEFAULT_TRANSLATION.